### PR TITLE
Add createContainer function for victory

### PR DESCRIPTION
**Pull request by ibarrae**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/victory/index.d.ts [hunk 1]
```diff
@@ -2424,4 +2426,24 @@ declare module "victory" {
...
+   * It allows you to effectively combine any two of the following containers:
+   * VictoryBrushContainer, VictoryCursorContainer, VictorySelectionContainer,
+   * VictoryVoronoiContainer, or VictoryZoomContainer.
+   * @param c1 : "brush" | "cursor" | "selection" | "voronoi" | "zoom"
+   * @param c2 : "brush" | "cursor" | "selection" | "voronoi" | "zoom"
+   */
+  export function createContainer<V, W>(
+    c1: ContainerType,
+    c2: ContainerType
+  ): React.FunctionComponent<V & W>;
```
Comment (allreadyExisted) [inline types/victory/index.d.ts]: Why not React.ComponentType?

File: types/victory/tslint.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
         "no-declare-current-package": false,
         "no-empty-interface": false,
         "no-single-declare-module": false,
-        "strict-export-declare-modifiers": false
+        "strict-export-declare-modifiers": false,
+        "no-unnecessary-generics": false
```
Comment (allreadyExisted) [inline types/victory/tslint.json]: Why disable rule?

File: types/victory/index.d.ts (hunk 1 above)
Comment (allreadyExisted) [reply types/victory/index.d.ts]:
```suggestion
  ): React.ComponentType<V & W>;
```

File: types/victory/index.d.ts [hunk 3]
```diff
@@ -5,6 +5,8 @@
 //                 Krzysztof Cebula <https://github.com/Havret>
 //                 Vitaliy Polyanskiy <https://github.com/allreadyExisted>
 //                 James Lismore <https://github.com/jlismore>
+//                 Stack Builders <https://github.com/stackbuilders>
+//                 Esteban Ibarra <https://github.com/ibarrae>
```
Comment (allreadyExisted) [inline types/victory/index.d.ts]: Who are Stack Builders. Is this your company? I apologize for this question.

Comment (ibarrae) [reply types/victory/index.d.ts]: Yeah, its also the fork from the changes are being made.

File: types/victory/index.d.ts (hunk 1 above)
Comment (ibarrae) [reply types/victory/index.d.ts]: Thanks for the tip @allreadyExisted 👍

**Review by alreadyExisted — APPROVED** [review]

**Pull request merged**